Server/BackEndMonitoringServer/include/CLogicalDiskStatusLifeCycle.h

```diff
@@ -0,0 +1,57 @@
+#pragma once
+
+#include "CHardwareInfoLifeCycle.h"
```

Comment:
Think about how to reduce the number of includes or move all of them in cpp

Reply:
I could move them into cpp. Visual studio don't see them in .h, if I do so - error during compilation

Reply:
Fixed, [commit](https://github.com/ITA-Dnipro/BackEndMonitoring/commit/28775d4e4e211a6985cecfb05174a510800a77d4#diff-0544ae2661f9747e70ea10c500758935c3f7d93fc11a7c69c55cadb18011b360)